cpp/oneapi/dal/algo/covariance/test/compute_parameters.cpp

```diff
@@ -74,18 +83,41 @@ TEMPLATE_LIST_TEST_M(covariance_params_test,
                      "[covariance][params]",
                      covariance_types) {
     SKIP_IF(this->not_float64_friendly());
+    using Float = std::tuple_element_t<0, TestType>;
+    using Method = std::tuple_element_t<1, TestType>;
+    const bool assume_centered = GENERATE(true, false);
+    INFO("assume_centered=" << assume_centered);
+    const bool bias = GENERATE(true, false);
+    INFO("bias=" << bias);
+    const cov::result_option_id result_option =
+        GENERATE(covariance::result_options::means,
+                 covariance::result_options::cov_matrix,
+                 covariance::result_options::cor_matrix,
+                 covariance::result_options::cor_matrix | covariance::result_options::cov_matrix,
+                 covariance::result_options::cor_matrix | covariance::result_options::cov_matrix |
+                     covariance::result_options::means);
+    INFO("result_option=" << result_option);
+
+    auto cov_desc = covariance::descriptor<Float, Method, covariance::task::compute>()
+                        .set_result_options(result_option)
+                        .set_assume_centered(assume_centered)
+                        .set_bias(bias);
 
     const te::dataframe input =
         GENERATE_DATAFRAME(te::dataframe_builder{ 500, 40 }.fill_uniform(-100, 100, 7777),
                            te::dataframe_builder{ 1000, 20 }.fill_uniform(-30, 30, 7777),
                            te::dataframe_builder{ 10000, 100 }.fill_uniform(-30, 30, 7777),
                            te::dataframe_builder{ 100000, 20 }.fill_uniform(1, 10, 7777));
+
+    INFO("num_rows=" << input.get_row_count());
```

Comment:
is it still necessary?

Reply:
do you mean the print? it allows to easier understand which dataset size failed in case of failure.